### DISCUSSION TITLE
Question: export topo maps level

### DISCUSSION THREAD

**Discussion by 321C** [johnkeel-thork/DMD2#98] [Q&A]
Hello João,
Is there any way to export topo maps levels of difficulty?
It is really pleasant on the field, and would be really useful to plan a trip
Thanks for your help!

**Comment by johnkeel-thork** [comment]
Hello
When you say "Export" what do you mean exactly?
Export into which format?

**Reply by 321C** [reply]
Whatever, I can work on it on my side, it could be .csv, .gpx, depending on what's easier
The thing is I'd like to plan trips knowing what path is ok, and what is too much difficult, on openstreetmap for example
